lib/cn/cndb.c

```diff
@@ -1109,10 +1111,11 @@ merr_t
 cndb_blkdel(struct cndb *cndb, union cndb_mtu *mtu, u64 txid)
 {
     struct blk_list blks;
-    u32             bx;
-    merr_t          err = 0;
-    u64 *           p;
-    u32             c;
+    uint32_t bx;
+    merr_t err = 0;
+    uint64_t *p;
+    uint32_t c;
+    uint64_t hoid;
```

Comment:
I'd put bx and c on same line to reduce both stack usage and LOC...

Reply:
Is this a similar concept to struct packing? Not sure how it relates to stack usage.

Reply:
Yeah.  I think (but do not know for certain) that local vars in C are laid out on stack in order give (or opposite order).  Damnit, now I have to go research it...

Reply:
So, turns out, the C spec 6.9.1p9 says the layout of storage is unspecified.  So it may be that the compiler gathers small variables to try and pack them within a cache line, and sticks arrays at the end/top/bottom.  But could do something unexpected to support address-space randomization.  I guess all bets are off...

Reply:
Given how we (not me because I hate it) try to conform to this c90 "declare at beginning of scope shenanigans", is there anyway to get the compiler to emit a warning when we write code that doesn't conform?